modules/serverless-config-replicas.adoc

```diff
@@ -0,0 +1,49 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-HA.adoc
+
+[id="serverless-config-replicas_{context}"]
+= Configuring high availability replicas on {ServerlessProductName}
+
+HA functionality is available by default on {ServerlessProductName} for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components. These components are configured with two replicas by default.
```

Comment:
I think we are supposed to spell out acronyms first in each module. 
_High availability (HA) functionality_